### PR TITLE
Prepare 0.68

### PR DESCRIPTION
**Pull request by guy-maurel**
and repair count_the_options.sh which was broken because of the changes at options.*

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+  - sp_return_brace                      Aug 21
+  - align_assign_decl_func               Jul 24
+  - sp_brace_brace                       Jul 23
+  - sp_after_decltype                    Jul 22
+  - align_right_cmt_same_level           Jul 14
+  - nl_oc_mdef_brace                     Jul  8
+  - mod_enum_last_comma                  Jul  8
+  - sp_type_ellipsis                     Jun 15
+  - indent_continue_class_head           May 30
+  - sp_decltype_paren
```
Comment (mwoehlke-kitware) [inline ChangeLog]: I think this is my bad, but there are a couple options (`sp_paren_ellipsis`, `sp_sizeof_ellipsis`, `sp_sizeof_ellipsis_paren`) that got added under 0.67 that actually should be here (apparently).

BTW, no date for `sp_decltype_paren`? (Should be Jul 21: 0f34dcb5d4ed... or Jul 24 if you go by merge date?)

Comment (guy-maurel) [reply ChangeLog]: Thanks.

File: release-steps.txt [hunk 2]
```diff
@@ -28,9 +28,9 @@ Change the date at the first line of this document.
       At line * highly configurable
       TODO: (automat for this) Change the values of options and version.
 
-2b. Update the option count in documentation/htdocs/index.html
-   Use script/count_the_options.sh to count the options.
+2b. Update the option count in documentation/htdocs/index.html .
    Check the number of options at ./tests/cli/output/help.txt
+   or use uncrustify --help .
```
Comment (mwoehlke-kitware) [inline release-steps.txt]: :+1: Thanks!

File: ChangeLog (hunk 1 above)
Comment (mwoehlke-kitware) [reply ChangeLog]: > There are a couple options that got added under 0.67 that actually should be here (apparently).

Nudge. LGTM otherwise.

**Pull request merged**